module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -205,8 +205,10 @@ public function homeAction()
                     if (
                         $this->params()->fromPost('processLogin')
                         && $this->inLightbox()
-                        && !$this->hasFollowupUrl()
+                        && (!$this->hasFollowupUrl()
+                        || $this->followup()->retrieve('lightbox') === false)
```

Comment:
As far as I can tell, 'lightbox' is a new value introduced by this PR. Is that intended? If so, can we give it a more descriptive name? I think that would help clarify what is going on here.

Also, does this value ever get cleared once set? Do we need to add logic to clearFollowupUrl() to get rid of it along with the other followup-related values?

Reply:
Renamed and clearing added.